A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml

```diff
@@ -882,6 +882,21 @@
             </Requires>
         </Sound>
 
+        <!-- Altitude deviation warning =====================================================================-->
+                <Sound WwiseData="true" WwiseEvent="cchordloop" NodeName="WIPER_BASE_L" ViewPoint="Inside" LocalVar="A32NX_ALT_DEVIATION" Continuous="true">
```

Comment:
@hotshotp to confirm the XML sound structure.

Reply:
Node name should be PEDALS_LEFT other wise good